### ISSUE TITLE
[UPDATE] For those who arrive at this repository to build with these tools.

### ISSUE THREAD

**Issue by markelwin** [mantyni/Multi-object-detection-lego#2]
For those who arrive at this repository to build I want to put this message to invite continuation of this project and collaboration with the rest of the industry. This repository is listed on several prominent LEGO and LEGO/ML field lists and is one of only two or three that are regularly included in these lists of resources for AI/ML and Blender engineers searching for tools to render LEGOs in Blender from the per-brick constituency on upwards. This repository includes useful software for camera positioning, yet its Blender files are no longer compatible with later versions of Blender, and so this message is for readers to reach out to work on further projects and to resolve questions about using these tools. My contact information can be found by searching for me online or by commenting below.

**Comment by mantyni** [comment]
Hello, it's great to hear that repository is being useful! I am still more than willing to support the repository going forward so please do post the issues here and I will get back to them as soon as I can. 

I will try out the later Blender version and update the repository to ensure it is fully compatible. It would be very helpful if you could share what specifically was not working with newer Blender.

**Comment by markelwin** [comment]
Wonderful to hear from you and connect. My primary intention over the next several weeks will be to learn how to import more shapes and obj shapefiles into the composition engine, and then work to understand how to apply basic constraints to the positional interfacing of individual components (bricks). Please feel free to reach out to me more directly privately if you want to work on collaborative elements of this project or further deep learning projects more thoroughly and broadly.

**Comment by markelwin** [comment]
I took the initiative to explore teaching myself the basics of Blender bpy this week so that I could construct a Blender environment for rendering LEGOs for realistic machine learning. I would love for you to help me continue to develop this.

I took this initiative to consolidate into one location a textbook of numerous introductory topics that introductory Blender Python and machine learning individuals want to know: rigid body physics, materials, mesh calculations, placement of multiple objects in an environments, rendering, and the beginning of resource management and processing. I found over and over again that searches for Blender `'bpy' refused to turn up single-source answers to these questions with consistent computer science paradigms, and so this new repository should drastically reduce the learning curve for machine learning individuals because these are common skills in machine learning. I hope the repository becomes the one-stop textbook for entry into Blender Python for the machine learning and LEGO communities, and I will be staffing the issues and reaching out to other community members as I develop out its utilities over the coming months and years. I would love for you to move your focus over to this new repository to help me build the project better, and to help any individuals that land on this page searching for Blender Python or machine learning to find the repository and reach out to me. There are vastly more individuals in this community than the resources currently seem to suggest, and I want to grow out these communities.

https://github.com/markelwin/LEGO.Emulation.Environment
